cosmo_tester/framework/fixtures.py

```diff
@@ -89,18 +89,26 @@ def distributed_installation(cfy, ssh_key, module_tmpdir, attributes, logger,
     If request.param has a 'sanity' value: 6 nodes are built (1 DB 1 Queue
     3 managers and an AIO manager)
 
+    If request.param has a 'cluster_upgrade' value: 8 nodes are built (1 DB 1
+    Queue 3 managers and 3 old AIO managers will be sent multiple times with...
+    ...'cluster_version' testing the cluster upgrade from 4.6 to later versions
```

Comment:
better to point to a const var, because we will forget to update this doc

Reply:
4.6 won't change, it's from old cluster to new cluster upgrade
I updated in the doc